src/Sentry/Internal/PlatformIntegration.uwp.cs

```diff
@@ -0,0 +1,54 @@
+#if WINDOWS_UWP
+using System;
+using System.Linq;
+using System.Runtime.ExceptionServices;
+using System.Security;
+using Sentry.Protocol;
+using Windows.UI.Xaml;
+
+namespace Sentry.Internal
+{
+    internal class PlatformIntegration : IInternalSdkIntegration
+    {
+        private SentryOptions _options;
+        private IHub? _hub;
+
+        public void Register(IHub hub, SentryOptions options)
+        {
+            _options = options;
+            _hub = hub;
+            //Sentry default Logger doesn't work with UWP.
+            options.AddEventProcessor(new PlatformEventProcessor(options));
+            var uwpApplication = Application.Current;
+            uwpApplication.UnhandledException += Handle;
+
+        }
+
+        public void Unregister(IHub hub)
+        {
+            _options.EventProcessors = _options.EventProcessors?.Where(p => p.GetType() != typeof(PlatformEventProcessor)).ToArray();
+            _hub = null;
+            var uwpApplication = Application.Current;
+            uwpApplication.UnhandledException -= Handle;
```

Comment:
Unsubscribe first, so events stop coming in..

I'm not a fan of swapping out this list mid-flight. It won't throw a "enumeration changed during enumeration" error but I'm not sure it's always safe to do this since multiple threads will be reading this list, each time any `CaptureX` is called.

```suggestion
            var uwpApplication = Application.Current;
            uwpApplication.UnhandledException -= Handle;
            _options.EventProcessors = _options.EventProcessors?.Where(p => p.GetType() != typeof(PlatformEventProcessor)).ToArray();
            _hub = null;
```

Also you might be unsubscribing to a different Application? Can `Current` ever change?
I believe you should hold a reference to the `Application` you've subscribed to. You'd anyway avoid GC to it so there's no worry about leaking it.